ssh-utils/ssh-pubkey.ps1

```diff
@@ -5,14 +5,14 @@
 
 if (!(Test-Path -Path "$Env:USERPROFILE/.ssh"))
 {
-    New-Item -Path "$Env:USERPROFILE/.ssh" -ItemType Directory
+    New-Item -Path "$Env:USERPROFILE/.ssh" -ItemType Directory -Force
     #& icacls "$Env:USERPROFILE/.ssh" /inheritance:r
     #& icacls "$Env:USERPROFILE/.ssh" /grant:r "$Env:USERNAME":"(F)"
 }
 
 if (!(Test-Path -Path "$Env:USERPROFILE/.ssh/config"))
 {
```

Comment:
This one is not a directory. We don't want `-Force` here.

Reply:
so you dont want "-Force" anywhere in this file